validation/create.go

```diff
@@ -70,14 +70,13 @@ func main() {
 
 		if err == nil {
 			state, err := r.State()
+			t.Ok(err == nil && state.ID == c.id, "'state' MUST return the state of a container")
 			if err == nil {
-				t.Ok(state.ID == c.id, "'state' MUST return the state of a container")
 				t.YAML(map[string]string{
 					"container ID": c.id,
 					"state ID":     state.ID,
 				})
 			} else {
-				t.Skip(1, "'state' MUST return the state of a container")
```

Comment:
This effectively rolls back part of d8d23968 (#564), but I didn't motivate the skip then, and can't remember why I'd felt it was a good idea.  I'm fine returning to to stricter handling like you've done here.